### DISCUSSION TITLE
파이썬으로 카카오쪽을 통해 예약 가능한 매크로 공유드립니다.

### DISCUSSION THREAD

**Discussion by changdoc** [changdoc/naver-vaccine-macro#90] [Announcements]
SJang1님께서 https://github.com/changdoc/naver-vaccine-macro/discussions/84 에서 설명해주셨는데요.

네이버 예약 신청 페이지에서 새로고침하는것 보다 확률이 높을 것 같아 공유드립니다.

https://github.com/SJang1/korea-covid-19-remaining-vaccine-macro

**Comment by MKK425** [comment]
프로그램까지 다 깔고 좌표를 입력해야하는데 개발자도구에서 좌표를 어디서 보는지 모르겠는데 혹시 알려주실 수 있을까요?

**Reply by jil8885** [reply]
구글 맵에서 검색할 범위를 사각형으로 정하고 왼쪽 위와 오른쪽 아래 부분의 경위도가 x,y 좌표입니다.

**Comment by presso22** [comment]
죄송한데 여러분 대체 graphql result가 어디에 있는건지 알수있을까요.. 도저히 못찾겠네요

**Reply by changdoc** [reply]
https://github.com/changdoc/naver-vaccine-macro/issues/4
